css/style.css

```diff
@@ -1,5 +1,5 @@
 .hero{
-    background-image: linear-gradient(0deg, rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(./assets/hero.jpg);
+    background-image: linear-gradient(0deg, rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(../assets/hero.jpg);
```

Comment:
Just a suggestion, you can use GitHub raw URL here so it won't have a dependency on folder structure